dashboard/test/widgets/task_overlay_test.dart

```diff
@@ -373,51 +373,8 @@ void main() {
     expect(find.text(TaskOverlayContents.rerunSuccessMessage), findsNothing);
   });
 
-  testWidgets('failed rerun shows error snackbar message', (WidgetTester tester) async {
```

Comment:
We should then be migrating this test to validate error brook surfaces the error

Reply:
Thanks a lot for taking the time to review the code! I tried various approaches but was not very successful in getting failed rerun tests to work. My current approach is (https://gist.github.com/XilaiZhang/2a962269c010d1650f4915de3f87f355): I tried to wrap the test widget with a errorBrook inside a valueProvider of buildState, but there is overflow and the super.disposal() from errorBrook throws an exception. I have tried other approaches such as borrowing code from build_dashboard_page.test but they create more errors. Could you provide some guidance on how I should get errorBrooks incorporated into my test? Thank you!

Reply:
I added some comments. I recommend taking a look at the error brook tests to see how they work.